### DISCUSSION TITLE
Is it possible to restart a livestream without restarting the container?

### DISCUSSION THREAD

**Discussion by java007md** [mrlt8/docker-wyze-bridge#797] [Q&A]
I occasionally encounter a problem with a camera dropping off the net due to wifi interference (most often from microwave). The streams saving recordings and snapshots recover once the interference clears, but livestreaming to youtube in particular seems to require a restart to get the stream going again.

Is is currently possible to restart only the livestream process, without restarting the container as a whole?

If not, would that be feasible to implement?

**Comment by mrlt8** [comment]
hmm, it should restart the livestream at the same time. Does `http://localhost:5000/restart/rtsp_server` help restart the streams?

Could also set `DEBUG_FFMPEG=true` to see if there are any errors with the RTMP stream.

**Comment by java007md** [comment]
Thanks. `http://localhost:5000/restart/rtsp_server` does restart the stream, at least in my testing so far. Nothing really jumped out at me with the `DEBUG_FFMPEG=true` setting, but I'll keep watching. It is tempting to just set up a cron job to restart the rtsp server on a regular schedule, especially when the stream is most likely to be interrupted.
